public/app/plugins/datasource/loki/querybuilder/components/LabelBrowserModal.tsx

```diff
@@ -48,7 +49,14 @@ export const LabelBrowserModal = (props: Props) => {
   };
 
   return (
-    <Modal isOpen={isOpen} title="Label browser" onDismiss={onClose}>
+    <Modal
+      isOpen={isOpen}
+      title="Label browser"
+      onDismiss={onClose}
+      className={css`
+        width: 1500px;
```

Comment:
Have you considered setting this up with relative value (`%` or `vw`)? Setting this to `1500px` can cause problems if the screen is smaller. But if you set `~80vw`/`80%`, it is going to look correct/same at all screens.

Reply:
@ivanahuckova i did consider this however there are two reasons i chose the px value.
- the current value for modal width is [px](https://github.com/grafana/grafana/blob/b365eb00220d4137e779b8468cb41286fe6a7d70/packages/grafana-ui/src/components/Modal/getModalStyles.ts#L20) so there is consistency
- i found in this case using the px value actually works better on smaller screen sizes (see images)

on the left is the `px` on the right is `%/vw`. i found that the `px` works better because on a smaller device it seems to just use 100% width. whereas using a `%` or `vw` it will always use that value meaning you are losing valuable space.

sorry if thats a bad explanation hopefully you can see what i mean from the images.
<div>
<img width="49%" alt="Screenshot 2022-12-12 at 10 27 41" src="https://user-images.githubusercontent.com/34524710/207023748-427e6d27-cfb2-48ff-bdc0-4f66ed4b0f41.png">
<img width="49%" alt="Screenshot 2022-12-12 at 10 27 50" src="https://user-images.githubusercontent.com/34524710/207023772-e1125c5d-db8c-43d7-a69c-41db7e41f2b8.png">
</div>

Reply:
looking into this a bit more, i think i do prefer using the `% / vw` value. it works better on most screens, the only downside is what i mentioned above where on a very small screen you waste a little space on the edges. i will change this now

Reply:
@svennergr ohh, idk why but i totally forgot that existed 😅